TrkDiag/inc/TimeClusterInfo.hh

```diff
@@ -22,6 +22,9 @@ namespace mu2e {
     Float_t _minhtime, _maxhtime; // min and max cluster hit time
     Float_t _maxover; // max overlap with another time cluster this event
     XYZVec _pos; // average position of cluster
+    Float_t _posx; // average position, S Middleton	
```

Comment:
Why do you repeat the contents of the XYZVec as separate floats?

Reply:
this maybe unnecessary I think I will just remove these additions if the code doesnt throw any exceptions to it as I think they are not really needed